### PR TITLE
Scale Text Correctly

### LINKED ISSUE #1497: Manim write text out of the window
### Describe the error
When I use the Scene TextExample found [here](https://3b1b.github.io/manim/getting_started/example_scenes.html), only part of the text show up. I need to zoom out to see everything, and when I save the video, I can only see like I haven't zoom out.

### Code and Error
**Code**:
[This](https://3b1b.github.io/manim/getting_started/example_scenes.html) Scene TextExample.

**Error**:
Can't see some words.

### Environment
**OS System**: Windows 10
**manim version**: Release 1.0.0
**python version**: 3.9.4

### LINKED ISSUE #1533: Tex text not the correct size
### Describe the error
When running the code, the size doesn't match the one expected (and quite a bit)

### Code and Error
**Code**:
```
#Made by JsonV on the Manim Discord
class NumGenerator(Scene):
    def construct(self):
        num_gen_tex = Tex("Number Generator: ").to_edge(UL)
        self.play(Write(num_gen_tex))

        i = 0
        nums = []
        animations = []
        for k in range(10):
            a = np.random.normal(loc=100, scale=15)
            num = DecimalNumber(num_decimal_places=0).set_value(a)
            num.next_to(num_gen_tex, RIGHT)
            nums.append(num)
            animations.append(Write(num))
            animations.append(num.animate.to_edge(DL).shift(RIGHT*i))
            i += 1.3

        for animation in animations:
            self.play(animation)
        for num in nums:
            num.set_fill(ORANGE)
        nums[0].set_fill(BLUE)

        self.wait()
```

### Output
**Result Expected**
https://user-images.githubusercontent.com/73761711/121065729-62826a00-c7c9-11eb-92c2-ee10d007166f.mp4

**Result Get**
https://user-images.githubusercontent.com/73761711/121065787-6f06c280-c7c9-11eb-8ae9-69710c64889d.mp4

### Environment
**OS System**: Windows 10
**manim version**: master 
**python version**: 3.9.5

### PR DESCRIPTION
**Pull request by naveen521kk**
## Motivation
`Text` wasn't scaled correctly previous with https://github.com/3b1b/manim/pull/1383

Closes #1497.
Closes #1533.

## Proposed changes
Change `TEXT_MOB_SCALE_FACTOR` value
Also deprecate `size` parameter

Also, now text isn't scaled when increasing font size
instead it is passed to the underlying enging
for handling. Though always a Text object is scaled
with a default value so that it fits the screen.

## Test

Locally I get the following with this scene. It would nice if others test this.
**Code**:
```py
from manimlib import *

ipsum = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
class TextExample(Scene):
    def fade_out_in(self, m , m1):
        self.play(FadeOut(m),FadeIn(m1),run_time=3)
    def construct(self):
        t = Text("Hello World")
        t1 = Text("Hello World \n new line")
        t2 = Text("Hello Cursive", font="Cursive")
        t3 = Text("Hello Cursive", font="Cursive", t2c = {'Hello':RED})
        t4 = Text("Hello Cursive", font="Cursive", t2c = {'Hello':RED,'Cursive':BLUE})
        t5 = Text("Hello Cursive", font="Cursive", t2c = {'Hello':RED,'Cursive':BLUE}, font_size=60)        
        t6 = Text(ipsum, font_size=30).scale(.7)
        self.add(t)
        self.fade_out_in(t,t1)
        self.fade_out_in(t1,t2)
        self.fade_out_in(t2,t3)
        self.fade_out_in(t3,t4)
        self.fade_out_in(t4,t5)
        self.fade_out_in(t5,t6)
```

**Result**:

https://user-images.githubusercontent.com/49693820/122561514-95e1b600-d05f-11eb-8367-54530dbe8abc.mp4

### CODE REVIEW

**Comment by naveen521kk** [comment]
@3b1b does this look good to you?

File: manimlib/mobject/svg/text_mobject.py [hunk 1]
```diff
@@ -46,7 +47,17 @@ class Text(SVGMobject):
...
-        self.lsh = self.size if self.lsh == -1 else self.lsh
+        if self.size:
+            warnings.warn(
+                "self.size has been deprecated and will "
+                "be removed in future.",
+                DeprecationWarning
+            )
+            self.font_size = self.size
+        if self.lsh == -1:
+            self.lsh = self.font_size + self.font_size * 0.3
```
Comment (3b1b) [inline manimlib/mobject/svg/text_mobject.py:58]: This 0.3 should probably be a constant or parameter defined elsewhere.

Comment (naveen521kk) [reply manimlib/mobject/svg/text_mobject.py]: Done https://github.com/3b1b/manim/pull/1553/commits/5765ab9055598b756c448e473ad7c594e1cb406a

**Comment by 3b1b** [comment]
Looks good, thanks!

**Pull request merged**